flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/table/BigQueryDynamicTableSink.java

```diff
@@ -91,12 +98,12 @@ public SinkRuntimeProvider getSinkRuntimeProvider(Context context) {
         // Set the logical type.
         ((RowDataToProtoSerializer) sinkConfig.getSerializer()).setLogicalType(this.logicalType);
         // Get the Datastream-API Sink.
-        return SinkV2Provider.of(BigQuerySink.get(this.sinkConfig, null));
+        return SinkV2Provider.of(BigQuerySink.get(this.sinkConfig, null), this.parallelism);
```

Comment:
We should not send parallelism if it was not set by the user. Replace implementation (here, and in `copy`) to call original method if parallelism is not set, and use method with parallelism only if it was set by the user

Reply:
Fixed.